### ISSUE TITLE
Be notified of the end of breaks

### ISSUE THREAD

**Issue by dkrmr** [TatriX/pomidor#9]
Title says it all. I'll be nice to have the same notification system, alert+sound, to mark the end of breaks.

**Comment by TatriX** [comment]
Sounds good. I'll try to add one.

**Comment by dkrmr** [comment]
Or just a hook.

**Comment by TatriX** [comment]
You can use existing hook currently:
```elisp
(defun my-pomidor-update-hook ()
  (let ((break-duration 30) ;; seconds
        (ellapsed (time-to-seconds (pomidor-overwork-duration))))
    (when (> ellapsed break-duration)
      (pomidor-play-sound-file-async pomidor-sound-overwork))))

(add-hook 'pomidor-update-hook #'my-pomidor-update-hook)
```

**Comment by dkrmr** [comment]
That doesn't works. It plays the overwork sound every 30 seconds :/

**Comment by TatriX** [comment]
See `pomidor-default-alert`.
You may want to add `alert` call to your hook
```elisp
(alert (format "Work!\Break: [%s]"
                   (format-time-string "%H:%M:%S" (pomidor-break-duration) t))
           :severity 'normal
           :icon pomidor-icon
           :title "Pomidor"
           :category 'pomidor)
```

**Comment by vfinotti** [comment]
I am experiencing the same problem. This issue is caused by the `pomidor-break-duration` function, that returns a big number when `:break` is `nil`. I have describe the issue and presented a possible fix in pull request #11.

**Comment by TatriX** [comment]
Closed in https://github.com/TatriX/pomidor/commit/5780ce2aaa98882a79e92f5eefaaf10fd8dbee3a
